docs/ARCHITECTURE.md

```diff
@@ -108,7 +108,7 @@ replace CoreData with any other database. Key notes:
 Our Networking framework offers a Swift API around the WooCommerce's RESTful endpoints. In this section we'll do a walkthru around several
 key points.
 
-
+More on [Networking](NETWORKING)
```

Comment:
`(NETWORKING.md)` instead of `(NETWORKING)`. Links must contain the full file name to work correctly.